3.12/graphs-smart-graphs-getting-started.md

```diff
@@ -6,17 +6,68 @@ title: ArangoDB SmartGraphs - Getting Started
 Getting started
 ---------------
 
-SmartGraphs **cannot use existing collections**. When switching to SmartGraph from an
-existing dataset you have to import the data into a fresh SmartGraph.
+SmartGraphs **cannot use existing collections**. When switching to SmartGraph
+from an existing dataset you have to import the data into a fresh SmartGraph.
 
 All collections that are being used in SmartGraphs need to be part of the same
-`distributeShardslike` group. The `smartGraphAttribute` and the number of shards are immutable.
-The `smartGraphAttribute` attribute is used to inform the database how to shard data and, as a 
-consequence, all vertices must have this attribute. The `_from` and `_to` attributes that 
-point _from_ one document _to_ another document stored in vertex collections are set by
-default, following the same smart sharding pattern. 
-
-## Create a SmartGraph
+`distributeShardslike` group. The `smartGraphAttribute` and the number of
+shards are immutable.
+The `smartGraphAttribute` attribute is used to inform the database how to shard
+data and, as a consequence, all vertices must have this attribute. The `_from`
+and `_to` attributes that point _from_ one document _to_ another document
+stored in vertex collections are set by default, following the same smart
+sharding pattern.
+
+## Create a SmartGraph using the Web Interface
+
+The Web Interface (also called Web UI) allows you to easily create and manage
+SmartGraphs. To get started, follow the steps outlined below.
+
+1. In the main page of the Web Interface, go to the left sidebar 
+   menu and select the **Graphs** tab.
+2. To add a new graph, click **Add Graph**.
+3. In the **Create Graph** dialog that appears, select the
+   **SmartGraph** tab.
+4. Fill in all the following fields:
+   - For **Name**, enter a name for the SmartGraph.
+   - For **Shards**, enter the number of shards the graph is using.
+   - Optional: For **Replication factor**, enter the total number of
+     desired copies of the data in the cluster.
+   - Optional: For **Write concern**, enter the total number of copies
+     of the data in the cluster required for each write operation.
+   - For **SmartGraph Attribute**, insert the attribute that is used to
+     smartly shard the vertices of the graph. Every vertex in your graph
+     needs to have this attribute. Note that it cannot be modified later.
```

Comment:
Unclear if this refers to the SmartGraph Attribute property or the attribute value, although I'm not sure if the latter can be modified later.

Reply:
The SmartGraph attribute is a required field and its value (i.e. regions) is immutable. Seems clear to me, do you have any suggestion to further improve this sentence?

Reply:
> Note that <ins>it</ins> cannot be modified later.

The reference isn't clear, it could refer to "Every vertex in your graph" or "SmartGraph Attribute". But since both are immutable, I guess it is fine.